cmd/hyperkube/hyperkube.go

```diff
@@ -55,7 +55,7 @@ func (hk *HyperKube) AddServer(s *Server) {
 // FindServer will find a specific server named name.
 func (hk *HyperKube) FindServer(name string) (*Server, error) {
 	for _, s := range hk.servers {
-		if s.Name() == name {
+		if s.Name() == name || s.AlternativeName == name {
```

Comment:
Yes, I thought about that as well, but I think the code changes are so small in any case it's worth to do it "the real way"